### PR TITLE
Added db ChnageLog for new indexes created as part of RDM-8485

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### **JIRA link (if applicable)**
https://tools.hmcts.net/jira/browse/RDM-8485

### **Change description**
Added new indexes that are already added in Prod as part of P1.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-RDM-8485.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<changeSet id="rdm-8485" author="suman.sreenivasan@hmcts.net">
+    <sql dbms="postgresql"
+         endDelimiter="\nGO"
+         splitStatements="true"
+         stripComments="true">
+        CREATE INDEX IF NOT EXISTS idx_case_data_deceaseddateofbirth ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{deceasedDateOfBirth}'))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_createdingapsfrom_dwpstate ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{createdingapsfrom,dwpstate}'))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_d8petitionerlastname ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{D8PetitionerLastName}'))));
+    </sql>
+</changeSet>
```
Comment (smathangi) [inline src/main/resources/db/changelog/db.changelog-RDM-8485.xml:15]: idx_case_data_appeal_appellant_identity_nino is missing

Comment (smathangi) [reply src/main/resources/db/changelog/db.changelog-RDM-8485.xml:15]: Please ask Alliu to provide above 4 index sql statements if he has. I guess, {createdingapsfrom,dwpstate} required proper syntax to pick up correctly.

Comment (smathangi) [reply src/main/resources/db/changelog/db.changelog-RDM-8485.xml:15]: CREATE INDEX idx_case_data_D8PetitionerLastName ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{D8PetitionerLastName}'))));
CREATE INDEX idx_case_data_createdInGapsFrom_dwpState ON public.case_data USING BTREE ((trim(upper((data #>> '{createdInGapsFrom}')))),(trim(upper((data #>> '{dwpState}')))));
CREATE INDEX idx_case_data_deceasedDateOfBirth ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{deceasedDateOfBirth}'))));
CREATE INDEX idx_case_data_appeal_appellant_identity_nino ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{appeal,appellant,identity,nino}'))));

Comment (sumansreenivasan) [reply src/main/resources/db/changelog/db.changelog-RDM-8485.xml:16]: Will added these changes.

**Pull request merged**